spec/ldclient_spec.rb

```diff
@@ -23,246 +24,32 @@
     JSON.parse(data, symbolize_names: true)
   end
 
```

Comment:
These tests certainly need to be re-added. As part of the refactoring, the client class itself has been broken up pretty substantially, so most of this can be moved to an `evaluation_spec.rb` file that tests the evaluation module.

However, I think it's worth reviewing this now-- the tests will be re-added in a follow up PR. This is passing the v2 integration harness so the evaluation part is pretty solid.